nautobot/core/testing/integration.py

```diff
@@ -15,7 +15,7 @@
 from nautobot.core import testing
 
 # URL used to connect to the Selenium host
-SELENIUM_URL = os.getenv("NAUTOBOT_SELENIUM_URL", "http://localhost:4444/wd/hub")
+SELENIUM_URL = os.getenv("NAUTOBOT_SELENIUM_URL", "http://selenium:4444/wd/hub")
```

Comment:
Why this change here?

Reply:
This was.. actually i should probably go  back and confirm itll work without this change. I had to make it for another repo, it may not be necessary here.